pkg/api/scylla/v1/types_cluster.go

```diff
@@ -331,6 +332,23 @@ type SchedulerTaskSpec struct {
 	// +kubebuilder:default:=3
 	// +optional
 	NumRetries *int64 `json:"numRetries,omitempty"`
+
+	// cron specifies the task schedule as a cron expression. It supports an extended syntax including @monthly, @weekly, @daily, @midnight, @hourly, @every X[h|m|s].
+	// +optional
+	Cron string `json:"cron,omitempty"`
+
+	// retryWait specifies an initial exponential backoff duration X[h|m|s].
+	// +optional
+	RetryWait string `json:"retryWait,omitempty"`
+
+	// timezone specifies the timezone of window and cron fields.
+	// +optional
+	Timezone string `json:"timezone,omitempty"`
```

Comment:
`Therefore, we ask that pointers always be used with optional fields that do not have a built-in nil value.`
https://github.com/kubernetes/community/blob/e9f78a/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
> that do not have a built-in nil value

""?

Reply:
that's not `nil` value, but a `zero` value

Reply:
for instance: https://github.com/scylladb/scylla-operator/blob/1520cf77e2bd089ad3497264dbd86196a4586f29/pkg/api/scylla/v1/types_cluster.go#L106

Reply:
Ok, I see there are counterexamples in our api as well, so that's not really an argument. I'll amend it

Reply:
done

Reply:
I agree mistakes were made :)

thanks for the update